docs/source/development/widget.md

```diff
@@ -251,9 +250,111 @@ See [Storybook](https://6.docs.plone.org/storybook) with available widgets.
 
 ## Write a new widget
 
-```{note}
-Please contribute to this section!
-```
+A custom widget is often used in Volto forms.
+To configure a new Volto widget follow these steps.
```

Comment:
a short list of steps before going into details would be nice.

Reply:
I don't understand why it would be nice and not redundant.